R/initModel.R

```diff
@@ -39,6 +40,7 @@ initModel <- function(config = NULL,
                       restart = NULL,
                       sendToSlurm = NULL,
                       slurmQOS = "default",
+                      tasksPerNode = 16,
                       tasks32 = FALSE) {
```

Comment:
I'm wondering if it was preferable to set these setting in the config instead of the function call. I expect one usually wants to set this run-specific. Otherwise, users have to remember to always pass the arguments when initialising the model. Sorry for not mentioning this earlier. I forgot that we had slurm arguments here already. What do you think?

Reply:
I was already thinking about this, too, but I'm not so sure about it. I think the 16 CPUs in priority might by far be the most commonly used setting - especially given that standby still does not work (I think). Having this in the function call is more flexible. Passing this via the config can be easier and faster if these configurations really are run-specific. I actually often use the same config for several runs. I guess it would also be possible to implement both - that you can pass SLURM settings via the function calls and if you don't, it is checked whether there are present in the config.

Reply:
I agree. Having both is probably the best.

Reply:
I implemented it now in such a way that both is possible, would be great if you could have another look.